clang/lib/Driver/Driver.cpp

```diff
@@ -8220,10 +8220,8 @@ Action *Driver::ConstructPhaseAction(
         ExtractBCFiles->addExtractColumnTform(FileTableTformJobAction::COL_ZERO,
                                               false /*drop titles*/);
         return ExtractBCFiles;
-
```

Comment:
I think @mdtoguchi's comment meant we don't need the 'else' construct. I think we still need the return statement after the 'if' construct.